docs/source/changelog.rst

```diff
@@ -1,6 +1,18 @@
 =========
 Changelog
 =========
+**Future**
+    * Changes
+        * Add `nltk-data.tar.gz` which contains the nltk corpuses we currently reference :pr:`31`
+        * Add that data to the `nltk` data path at import-time :pr:`31`
```

Comment:
I think we should make it clear that we're no longer downloading nltk data over the internet after installing the package

Reply:
Yep thanks will do